css/modern.css

```diff
@@ -1755,9 +1759,124 @@ table#collection-view-contents tbody tr td {
     background-color: var(--highlight) !important;
 }
 
+<<<<<<< HEAD
+
+/* Quizes */
+
+#s-assessment-view-submission-form .selected.correct  span.option-span {
+    background-color: var(--active) !important;
+    color: var(--text) !important;
+}
+
+
+#s-assessment-view-submission-form .incorrect span.option-span.legacy-true-false-question, #s-assessment-view-submission-form .incorrect span.option-span.legacy-multiple-choice-question {
+    background-color: var(--highlight) !important;
+    color: var(--text) !important;
+}
+
+/* California Assesment */
+
+.ca-assessment-delivery-take-assessment {
+    background-color: var(--background-color) !important;
+}
+
+.lrn-assess > div {
+    background-color: var(--background-color) !important;
+}
+
+.lrn.lrn-assess .lrn-right-region .right-wrapper .menu-overlay {
+    background-color: var(--border-color) !important;
+    color: var(--text) !important;
+}
+
+
+.lrn .pagination>li>a, .lrn .pagination>li>span {
+    background-color: var(--border-color) !important;
+
+}
+
+.lrn .pagination>li>a, .lrn .pagination>li>span {
+    background-color: var(--border-color);
+    color: var(--text)!important;
+}
+
+.lrn.lrn-assess .lrn-horizontal-toc .pagination .lrn-assess-btn:focus, .lrn.lrn-assess .lrn-horizontal-toc .pagination .lrn-assess-btn:hover, .lrn.lrn-assess .lrn-horizontal-toc .lrn-assess-pagination .lrn-assess-btn:focus, .lrn.lrn-assess .lrn-horizontal-toc .lrn-assess-pagination .lrn-assess-btn:hover {
+    background: var(--border-color) !important;
+    color: #333 !important;
+}
+
+.lrn.lrn-assess .test-title-text {
+    color: var(--text) !important;
+}
+
+.lrn.lrn-assess .app-panel.review-screen .panel-content {
+    border: 1px solid var(--border-color) !important;
```

Comment:
Please use `border: var(--border-width) solid var(--contrast-border)` instead of what you have here
Don't use border-color